tsconfig.json

```diff
@@ -1,50 +1,27 @@
-// GENERATED BY UPKEEP
 {
   "extends": "./tsconfig.settings.json",
   "compilerOptions": {
     "noEmit": true,
     "baseUrl": ".",
     "skipLibCheck": false,
     "paths": {
-      "@coil/monorepo-upkeep": [
-        "packages/coil-monorepo-upkeep/src"
-      ],
-      "@coil/polyfill-utils": [
-        "packages/coil-polyfill-utils/src"
-      ],
-      "@coil/privacypass-sjcl": [
-        "packages/coil-privacypass-sjcl/src"
-      ],
-      "@coil/webpack-utils": [
-        "packages/coil-webpack-utils/src"
-      ],
-      "@dier-makr/annotations": [
-        "packages/dier-makr-annotations/src"
-      ],
+      "@coil/monorepo-upkeep": ["packages/coil-monorepo-upkeep/src"],
```

Comment:
run `yarn upkeep`, it should format these in the canonical format expected by CI

Reply:
pushed !

Reply:
Thanks! I was wondering what I need to do to fix this issue.